kata-deploy/README.md

```diff
@@ -1,22 +1,19 @@
 # kata-deploy
 
-
 - [kata-deploy](#kata-deploy)
-  * [Docker quick start](#docker-quick-start-)
+  * [Docker quick start:](#docker-quick-start-)
     + [Install Kata and configure Docker](#install-kata-and-configure-docker)
-    + [Run a sample workload](#run-a-sample-workload-utilizing-kata-containers)
+    + [Run a sample workload utilizing Kata containers:](#run-a-sample-workload-utilizing-kata-containers-)
     + [Remove Kata](#remove-kata)
-  * [Kubernetes quick start](#kubernetes-quick-start-)
+  * [Kubernetes quick start](#kubernetes-quick-start)
     + [Install Kata on a running Kubernetes cluster](#install-kata-on-a-running-kubernetes-cluster)
-    + [Run a sample workload](#run-a-sample-workload-)
-    + [Remove Kata from the Kubernetes cluster](#remove-kata-from-the-kubernetes-cluster-)
+    + [Run a sample workload](#run-a-sample-workload)
+    + [Remove Kata from the Kubernetes cluster](#remove-kata-from-the-kubernetes-cluster)
   * [kata-deploy details](#kata-deploy-details)
     + [Dockerfile](#dockerfile)
-    + [Daemonsets and RBAC](#daemonsets-and-rbac-)
-      - [runtime-labeler](#runtime-labeler-)
-      - [CRI-O and containerd kata installer](#cri-o-and-containerd-kata-installer-)
-    + [Kata cleanup](#kata-cleanup-)
-
+    + [Daemonsets and RBAC:](#daemonsets-and-rbac-)
+      - [Kata installer: kata-deploy](#kata-installer--kata-deploy)
+    + [Kata cleanup:](#kata-cleanup-)
```

Comment:
`[Kata cleanup](#kata-cleanup)` ?

Reply:
The `:` was being resolved to `-` in the TOC.  I removed the `:`s :)

Reply:
https://github.com/isaacs/github/issues/215 stings.

/me tries valiantly not to preach the benefits of reStructuredText which auto-generates TOCs... even on github... Wait... :smile: